sw/host/tests/manuf/personalize/src/main.rs

```diff
@@ -116,13 +116,15 @@ fn rma_unlock_token_export(opts: &Opts, transport: &TransportWrapper) -> Result<
     );
 
     // Issue an LC transition to RMA to verify unlock token.
+    // FIXME unsafe?
```

Comment:
this won't be unsafe if we fix the device binary as [suggested](https://github.com/lowRISC/opentitan/issues/19086#issuecomment-1693810502) I think